pom.xml

```diff
@@ -539,7 +539,7 @@
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
         <configuration>
-          <argLine>-XX:MaxMetaspaceSize=128m ${jacoco.agent.it.arg}</argLine>
+          <argLine>-XX:MaxMetaspaceSize=512m ${jacoco.agent.it.arg}</argLine>
```

Comment:
I picked 512m because that's the value currently in use for the (surefire) unit tests: see line 561 below. That is, unless I'm misreading this, it seems fcrepo already requires .5 GB for the tests (and has for some time).